ecosystem/sep-0024.md

```diff
@@ -316,48 +314,15 @@ When uploading data for fields specificed in [SEP-9](sep-0009.md), `binary` type
 
 ### Response
 
-There are several possible kinds of response, depending on whether the anchor needs more information about the user, how it should be sent to the anchor, and if there are any errors.
-
-The first response, the success response, is explained below. The other possible responses are shared with the deposit endpoint, and are explained in the [Deposit and Withdraw shared responses](#deposit-and-withdraw-shared-responses) section directly below.
-
-#### 1. Success: no additional information needed
```

Comment:
I agree we should remove this. Its not used by any known anchor and SEP-6 provides a non-interactive alternative. There was a PR that turned stale to do the same thing: https://github.com/stellar/stellar-protocol/issues/539